pkg/dashboard/resource/frontendspec.go

```diff
@@ -171,6 +173,21 @@ func (fsr *frontendSpecResource) resolveDefaultFunctionNodeSelector() map[string
 	return defaultNodeSelector
 }
 
+func (fsr *frontendSpecResource) getDefaultHTTPIngressHostTemplate() string {
+
+	// backwards compatibility with old helm charts that inject the ingress host template via envvar
+	defaultHTTPIngressHostTemplate := common.GetEnvOrDefaultString(
+		"NUCLIO_DASHBOARD_HTTP_INGRESS_HOST_TEMPLATE", "")
+	if defaultHTTPIngressHostTemplate != "" {
+		return defaultHTTPIngressHostTemplate
+	}
+
+	if dashboardServer, ok := fsr.resource.GetServer().(*dashboard.Server); ok {
+		defaultHTTPIngressHostTemplate = dashboardServer.GetPlatformConfiguration().Kube.DefaultHTTPIngressHostTemplate
+	}
+	return defaultHTTPIngressHostTemplate
```

Comment:
Doesn't matter too much (cause it will probably never happen) but I assume the opposite precedence should be applied, i.e. if both are configured, the new configuration "place" should apply and not the old one (env var)

Reply:
only if set of (current) latter course, but agree